v1/ao/layer.go

```diff
@@ -245,6 +259,8 @@ func (s *layerSpan) BeginProfile(profileName string, args ...interface{}) Profil
 // End a profiled block or method.
 func (s *span) End(args ...interface{}) {
 	if s.ok() {
+		globalTags := s.GetTrace().GetGlobalTags()
```

Comment:
Not sure if we want globalTags in exit events too? Since they are should already be in the entry event?